core/openapi/schemas/schemaLevel3.yaml

```diff
@@ -0,0 +1,132 @@
+oneOf:
+  - $ref: reference.yaml
+  - type: object
+    properties:
+      title:
+        type: string
+      multipleOf:
+        type: number
+        minimum: 0
+        exclusiveMinimum: true
+      maximum:
+        type: number
+      exclusiveMaximum:
+        type: boolean
+        default: false
+      minimum:
+        type: number
+      exclusiveMinimum:
+        type: boolean
+        default: false
+      maxLength:
+        type: integer
+        minimum: 0
+      minLength:
+        type: integer
+        minimum: 0
+        default: 0
+      pattern:
+        type: string
+        format: regex
+      maxItems:
+        type: integer
+        minimum: 0
+      minItems:
+        type: integer
+        minimum: 0
+        default: 0
+      uniqueItems:
+        type: boolean
+        default: false
+      maxProperties:
+        type: integer
+        minimum: 0
+      minProperties:
+        type: integer
+        minimum: 0
+        default: 0
+      required:
+        type: array
+        items:
+          type: string
+        minItems: 1
+        uniqueItems: true
+      enum:
+        type: array
+        items: {}
+        minItems: 1
+        uniqueItems: false
+      type:
+        type: string
+        enum:
+        - array
+        - boolean
+        - integer
+        - number
+        - object
+        - string
+      not:
+        oneOf:
+        - "$ref": "schemaFull.yaml"
+        - "$ref": "reference.yaml"
+      allOf:
+        type: array
+        items:
+          oneOf:
+          - "$ref": "schemaFull.yaml"
+          - "$ref": "reference.yaml"
+      oneOf:
+        type: array
+        items:
+          oneOf:
+          - "$ref": "schemaFull.yaml"
+          - "$ref": "reference.yaml"
+      anyOf:
+        type: array
+        items:
+          oneOf:
+          - "$ref": "schemaFull.yaml"
+          - "$ref": "reference.yaml"
+      items:
+        oneOf:
+        - "$ref": "schemaFull.yaml"
+        - "$ref": "reference.yaml"
+      properties:
+        type: object
+        additionalProperties:
+          oneOf:
+          - "$ref": "schemaFull.yaml"
+          - "$ref": "reference.yaml"
+      additionalProperties:
+        oneOf:
+        - "$ref": "schemaFull.yaml"
+        - "$ref": "reference.yaml"
+        - type: boolean
+        default: true
+      description:
+        type: string
+      format:
+        type: string
+      default: {}
+      nullable:
+        type: boolean
+        default: false
+      readOnly:
+        type: boolean
+        default: false
+      writeOnly:
+        type: boolean
+        default: false
+      example: {}
+      deprecated:
+        type: boolean
+        default: false
+      contentMediaType:
+        type: string
+      contentEncoding:
+        type: string
+      contentSchema:
+        $ref: "schemaLevel3.yaml"
+    patternProperties:
```

Comment:
Keep in mind: If we use this, we will loose swaggerhub capabilities.

Reply:
What are you referring to for this swaggerhub issue ?
Thanks.

Reply:
I am referring to patternProperties. This works only foor OpenAPI 3.1, which swaggerhub does not seem to support yet. I usually compile all the schemas into one OpenAPI document on swaggerhub to do sanity checks and to generate code.